packages/info-sprinkle/src/InfoSprinkle/InfoSprinkle.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { cx } from '@leafygreen-ui/emotion';
+import InfoWithCircleIcon from '@leafygreen-ui/icon/dist/InfoWithCircle';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import Tooltip from '@leafygreen-ui/tooltip';
+
+import { iconBaseStyles, iconThemeStyles } from './InfoSprinkle.styles';
+import { Align, InfoSprinkleProps, Justify } from './InfoSprinkle.types';
+
+export const InfoSprinkle = React.forwardRef<
+  HTMLSpanElement,
+  InfoSprinkleProps
+>(
+  (
+    {
+      darkMode: darkModeProp,
+      children,
+      baseFontSize,
+      ...rest
+    }: InfoSprinkleProps,
+    forwardRef,
+  ) => {
+    const { darkMode, theme } = useDarkMode(darkModeProp);
+    return (
+      <Tooltip
+        darkMode={darkMode}
+        baseFontSize={baseFontSize}
+        trigger={
+          <span
+            // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+            tabIndex={0}
+            ref={forwardRef}
+            className={cx(iconBaseStyles, iconThemeStyles(theme))}
+            data-testid="info-sprinkle-icon"
+          >
+            <InfoWithCircleIcon size={baseFontSize} aria-hidden />
```

Comment:
I can foresee folks writing in to ask if they can use other icons. What's our answer?

Reply:
Also, @alvenw I don't see anything in the guidelines about not using other icons. We can pre-empt developer requests by making sure designers don't use it incorrectly

Reply:
Our way of enforcing designers to only use the Info icon is that we made the Figma component with only Info icon without the ability to swap it for other icons. We can also add that to the guidelines though but it would be good to make sure developers aren't able to freely swap the icon